lib/condor.py

```diff
@@ -33,6 +33,7 @@
 import packages
 from render_files import render_files
 from tracing import as_span
+from transfer_sandbox import transfer_sandbox
```

Comment:
Could you please delete the `# pylint: disable=cyclic-import` statement on line 31 and make sure that pylint-via-pre-commit is good with this change?

Otherwise, it looks good.